libnopegl/src/block.c

```diff
@@ -197,15 +197,15 @@ int ngli_block_add_field(struct block *s, const char *name, int type, size_t cou
     return 0;
 }
 
-void ngli_block_field_copy(const struct block_field *fi, uint8_t * restrict dst, const uint8_t * restrict src)
+void ngli_block_field_copy_count(const struct block_field *fi, uint8_t * restrict dst, const uint8_t * restrict src, size_t max_count)
```

Comment:
In commit desc: pipeline -> pipeline_compat.

Reply:
done